upload/upload.go

```diff
@@ -26,7 +26,7 @@ func GetFile(r *http.Request) (multipart.File, *multipart.FileHeader, error) {
 		return file, fileHeader, nil
 	}
 	if err != nil {
-		l.Log.Info("Unable to find `file` or `upload` parts", zap.Error(err))
+		l.Log.Error("Unable to find `file` or `upload` parts", zap.Error(err))
```

Comment:
Since the handler is going to log this, it's best to just return the error.

Reply:
Fixed